cli/command/task_test.go

```diff
@@ -293,7 +301,7 @@ func TestParseOverrides(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	assert.Equal(t, len(output), len(expected))
+	assert.Equal(t, output, expected)
```

Comment:
params need to be reversed here, `assert.Equal(t, expected, output)`

Reply:
got it